### ISSUE TITLE
Shotgun pump/XM88 lever sound effect will not play if you press the action button too quickly

### ISSUE THREAD

**Issue by PolarTundraa** [RMC-14/RMC-14#5120]
## Description
As described in the title the sound effect just fails to play if you press the action button rapidly

**Reproduction**
shoot the pump shotgun or xm88 constantly while spamming the spacebar/unique action keybind, they will pump correctly and be made able to fire but the sound effect will not play

https://github.com/user-attachments/assets/225a9fa7-528d-44fa-afa5-928d11d6f2de

doesn't affect gameplay really other than immersion factors and knowing with certainty when exactly you have pumped the weapon

**Comment by Dutch-VanDerLinde** [comment]
Gonna try to fix this

**Comment by Dutch-VanDerLinde** [comment]
This was fixed
